jenkins/opensearch-dashboards/distribution-build.jenkinsfile

```diff
@@ -75,16 +75,18 @@ pipeline {
             }
             steps {
                 script {
-                    getRepositoryCommit(
-                        componentName: "${COMPONENT_NAME}",
-                        inputManifest: "manifests/${INPUT_MANIFEST}",
-                        outputFile: "commits.yml"
+                    buildInfoYaml(
+                        componentName: COMPONENT_NAME,
+                        inputManifest: "manifests/$INPUT_MANIFEST",
+                        outputFile: "buildInfo.yml",
```

Comment:
Can we rename this to something not containing camelCase, we will land on case-insensitive file systems. Maybe just `build.yml`?

Reply:
I am thinking the main purpose of this yaml file is to provide the build information (e.g. each stage status, start time & duration, the job ids of triggering tests) from each workflow. `build.yml` maybe a little confusing with the build manifest. WDYT @dblock

Reply:
Isn't this all already provided by exporting a jenkins job history?
https://stackoverflow.com/questions/52866480/export-build-history-of-jenkins-job